platforms/jvm/code-quality/src/main/java/org/gradle/api/plugins/quality/Checkstyle.java

```diff
@@ -315,13 +315,13 @@ public Property<Boolean> getEnableExternalDtdLoad() {
     @ReplacesEagerProperty(adapter = IsIgnoreFailuresAdapter.class)
     public Property<Boolean> getIsIgnoreFailures() {
         ProviderApiDeprecationLogger.logDeprecation(getClass(), "getIsIgnoreFailures()", "getIgnoreFailures()");
-        return getIgnoreFailuresProperty();
+        return getIgnoreFailures();
```

Comment:
@asodja, would it make sense to replace this manual adapter with the annotation-based declaration?

I presume it would be exactly like here, right?
https://github.com/gradle/gradle/pull/31187/files#diff-c482aabc3405144eb6808e676846c73e4cd87087b2d608a97387b8da5e41e89aR89-R95